src/style.css

```diff
@@ -0,0 +1,68 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:wght@400;700&display=swap');
+
+* {
+    box-sizing: border-box;
+    margin: 0px;
+    text-align: center;
+}
+
+body {
+    font-family: 'Noto Sans', 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+}
+
+h1 {
+    font-size: 45px;
+    padding: 20px;
+}
+
+h3 {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.header {
+    background-color:#6e5494;
+    text-align: center;
+    padding-bottom: 30px;
+    margin-bottom: 20px;
+}
+
+.submit-button {
+    border: none;
+    background-color: #f5f5f5;
+    margin-left: 10px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    padding-left: 10px;
+    padding-right: 10px;
+    border-radius: 5px;
+}
+
+.repo-section {
+    line-height: 40px;
+    font-weight: 200;
+}
+
+
+
+.result-item {
+    background-color: rgb(212, 193, 255);
+    width: 320px;
+    text-align: center;
+    margin: 15px;
+    padding: 20px;
+    display: inline-block;
+}
+
+.result-item > h3 > a {
+    color: inherit;
+}
+
+.result-item > h3 > a:hover {
+    opacity: 0.65;
+}
```

Comment:
card formatting is really nice, is there a reason you decided not to style the components within the components themselves?